services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_tasks.py

```diff
@@ -327,6 +373,8 @@ async def upsert_tasks_from_project(
                 user_id,
                 product_name,
                 conn,
+                rut_api,
+                is_wallet,
             )
             # get current tasks
             result = await conn.execute(
```

Comment:
@sanderegg can you please double check whether I do not need to change something in this section below? more specifically I can see there is an exlucsion_rule for "state" and "progress", so just want to be sure i do not need to add it there, thanks.

Reply:
I think that might be ok. just test what happens to the values when you move a node around

Reply:
@sanderegg so I have double-checked it and I think I need to add it there! Because when you change the instance for example from SMALL to MEDIUM, it needs to be updated in the table (currently it is not) So I will add it there, and please when you start to use it just think about whether there might be some corner cases (or we can discuss together).